frontend/src/app/lib/addon.js

```diff
@@ -138,6 +139,9 @@ function messageReceived(store, evt) {
       store.dispatch(addonActions.setInstalled(data.installed));
       store.dispatch(addonActions.setInstalledAddons(data.active));
       break;
+    case 'addon-self:installed':
+      console.log("ADDON_SELF_INSTALLED");
```

Comment:
I think this is the right place to resolve a promise indicating all the Test Pilot add-on event handlers are installed and it is safe to install the experiment, but I haven't been able to hook everything up properly yet.